csrankings-m.csv

```diff
@@ -979,6 +979,8 @@ Martin Henze,RWTH Aachen University,https://www.martinhenze.de/,nnpRKx4AAAAJ
 Martin Hitz,University of Klagenfurt,https://www.aau.at/team/hitz-martin,_Q8-XXAAAAAJ
 Martin Hoefer 0001,Goethe University Frankfurt,http://algo.cs.uni-frankfurt.de/~mhoefer,PBCXFY8AAAAJ
 Martin Hofmann 0001,LMU Munich,https://www.tcs.ifi.lmu.de/people/martin-hofmann,MuXVKIMAAAAJ
+Martin Höst,Malmö University,https://mau.se/en/persons/martin.host/,9HNtvo0AAAAJ

```

Comment:
Not in CS; also, see existing entry at Lund.

Reply:
Martin seems to have moved from Lund, as per [his LinkedIn profile](https://www.linkedin.com/in/martin-h%C3%B6st-bbab05/)

He is a part of the [Department of Computer Science and Media Technology](https://mau.se/en/about-us/faculties-and-departments/faculty-of-technology-and-society/department-of-computer-science-and-media-technology/).

He is found in the Software Engineering field, under said department. Since, SE is one of the categories in CSRankings, and he remains under the CS department of Malmo, I was under the impression that he is eligible for inclusion into CSRankings.

Please let me know if that is not the case, and I shall remove him.